subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3193,36 +3191,54 @@ static inline void event_len_prep(struct ll_conn *conn)
 
 		if (!conn->common.fex_valid ||
 		    (!feature_coded_phy && !feature_phy_2m)) {
-			rx_time = PKT_US(LL_LENGTH_OCTETS_RX_MAX, 0);
+			rx_time = MAX(PKT_US(LL_LENGTH_OCTETS_RX_MAX, 0),
```

Comment:
revert the MAX here as its applied in line 3233

Reply:
Almost, here we calculate the time required for a packet when using coded phy
In line 3233 we apply the Bluetooth spec minimum requirement for rx-time (and tx-time)

Reply:
PKT_US(PDU_DC_PAYLOAD_SIZE_MIN, 0)  is 328, hence do not need to do MAX twice, waste of CPU time due to branching

Reply:
There is no difference in code size either way; IMO as it is the code shows that there are 2 aspects in the length calculation; the compiler does its optimisation for reducing the code size